### ISSUE TITLE
Distro-specific icons show wrong icon for nerdfont

### ISSUE THREAD

**Issue by blackb1rd** [bhilburn/powerlevel9k#774]
The GH-758 have changed code wrong as below

      LINUX_ARCH_ICON                $'\uF300'              
      LINUX_DEBIAN_ICON              $'\uF302'              
      LINUX_UBUNTU_ICON              $'\uF30C'              
      LINUX_CENTOS_ICON              $'\uF301'              
      LINUX_COREOS_ICON              $'\uF30F'              
      LINUX_ELEMENTARY_ICON          $'\uF311'              
      LINUX_FEDORA_ICON              $'\uF303'             
      LINUX_GENTOO_ICON              $'\uF310'              
      LINUX_MINT_ICON                $'\uF304'             
      LINUX_MAGEIA_ICON              $'\uF306'              
      LINUX_OPENSUSE_ICON            $'\uF308'             
      LINUX_SABAYON_ICON             $'\uF313'              
      LINUX_SLACKWARE_ICON           $'\uF30A'              

To fix this issue we need to change code as @ryanoasis mention at #751 with following code change below:

      LINUX_ARCH_ICON                $'\uF303'              
      LINUX_DEBIAN_ICON              $'\uF306'              
      LINUX_UBUNTU_ICON              $'\uF31B'              
      LINUX_CENTOS_ICON              $'\uF304'              
      LINUX_COREOS_ICON              $'\uF305'              
      LINUX_ELEMENTARY_ICON          $'\uF309'              
      LINUX_FEDORA_ICON              $'\uF30A'             
      LINUX_GENTOO_ICON              $'\uF30D'              
      LINUX_MINT_ICON                $'\uF30E'             
      LINUX_MAGEIA_ICON              $'\uF310'              
      LINUX_OPENSUSE_ICON            $'\uF314'             
      LINUX_SABAYON_ICON             $'\uF317'              
      LINUX_SLACKWARE_ICON           $'\uF318'

**Comment by sambadevi** [comment]
@blackb1rd are you using the latest release of nerd-fonts? The mapping @ryanoasis mentioned seems to be WIP for the coming major version which isn’t released yet AFAIK.

**Comment by sambadevi** [comment]
@bhilburn this is kind of a duplicate of #756, when nerd-fonts will be pushed to the next major (probably 2.0) I will fix the mapping and also add the new added glyphs for the distros mentioned in #756

**Comment by blackb1rd** [comment]
> are you using the latest release of nerd-fonts?
I'm using the HEAD of master branch which's included the new mapping already.

**Comment by ryanoasis** [comment]
### TL;DR:
As others have already mentioned the _'Font Linux'_ codepoints have shifted: this is reflected in **Nerd Fonts** `master` and the soon to be released **Nerd Fonts** `2.0.0`.

Until **P9k** adjusts for `2.0.0` you will have breakage if you use the fonts from [`master` :warning: :construction:](https://github.com/ryanoasis/nerd-fonts/tree/master). You'd have to use the fonts from [`1.2.0` :pushpin:](https://github.com/ryanoasis/nerd-fonts/tree/1.2.0). :smiley: 

---

Yeah unfortunately we found it difficult to use the new codepoints from 'Font Logos' (formerly Font Linux) given the current way the patcher works (with a range of sequence). Their new glyphs were not just added to the current sequence but the entire order of the glyphs changed.

This and other reasons are why I decided to version the next release of Nerd Fonts as `2.0.0` (breaking changes - regarding these specific glyph codepoints). `master` on nerd-fonts is basically considered dev/bleeding edge but mostly is stable. 

As I [mentioned earlier](https://github.com/bhilburn/powerlevel9k/issues/410#issuecomment-286902856) on P9k the Nerd Fonts codepoints would be stable with the rare exception and in that case the major version would change to reflect. This is what has happened. Going forward we might try to adjust and be less reliant on these glyph sets and take what we want and put it where we want but that'll require quite a bit of work.

**Comment by sambadevi** [comment]
nerd-fonts `2.0.0` is out, I will add the code for now included distro-icons and create a PR to fix this issue and #756

**Comment by ryanoasis** [comment]
Just Fyi there was an issue. Had to pull the release. Need to rebuild all fonts

**Comment by sambadevi** [comment]
@ryanoasis do you know whether the mapping will change? If it stays like the current master version `2.0.0` nothing needs to change on our side

**Comment by ryanoasis** [comment]
Font Logos (Font Linux) is okay actually. The issue is with Octicons and Weather.

**Comment by bhilburn** [comment]
@ryanoasis - Thanks so much for weighing in, here! Your participation and guidance is very much appreciated =)

@blackb1rd - And thank you for spotting the issue and calling this out!

**Comment by bhilburn** [comment]
Closed by merge of #780
